platform-configs/msi-pro-z690-a-ddr5.robot

```diff
@@ -13,3 +13,18 @@ ${DEVICE_AUDIO2}=                   Raptorlake HDMI
 
 ${CPU_MAX_FREQUENCY}=               5200
 ${CPU_MIN_FREQUENCY}=               300
+
+
+${CPU_TESTS_SUPPORT}=                           ${TRUE}
```

Comment:
Probably pre-commit complains on lack of alignment.

Reply:
Also, does it belong to this MR?

If so, have we checked the other MSIs have these flags set as well?

Reply:
They do not. I figured it should be added as I needed to add these flags to check if the tests work now, and it was only performed on this single platform. Although you are probably right that he flags  should be either added everywhere, or nowhere...

Reply:
>Probably pre-commit complains on lack of alignment.

That also was a problem, although it was not listed explicitly like the issue with the "Boot System..." keyword, so I haven't noticed that, and so turns out I haven't had pre-commit installed.

@macpijan 
I think I have solved this issue for good by making pre-commit install itself automatically in every repo that uses it using git templates:
```
$ pre-commit init-templatedir ~/.git-template
$ git config --global init.templateDir ~/.git-template
```
The `~/.git-template` can be any directory.

Now when I clone any repository, the pre-commit hook is automatically installed thanks to the template.
Do you think we could add the instructions on how to setup this somewhere?

Reply:
> Do you think we could add the instructions on how to setup this somewhere?

https://github.com/3mdeb/workstation-config ?

Reply:
I also thought about this repository. I haven't worked with ansible before, so I will just issue an issue https://github.com/3mdeb/workstation-config/issues/53